package.json

```diff
@@ -23,15 +23,16 @@
   },
   "homepage": "https://github.com/vikpe/react-webpack-typescript-starter",
   "scripts": {
-    "build": "npm run clean-dist && webpack --mode production --config=configs/webpack/prod.js",
+    "build": "npm run clean-dist && webpack --config=tools/client-webpack/prod.js",
     "clean-dist": "rm -f -r -d dist",
     "lint": "npm run lint:ts && npm run lint:sass",
     "lint:ts": "tslint './src/**/*.ts*' --format stylish --force",
     "lint:sass": "stylelint ./src/**/*.scss",
-    "start": "npm run clean-dist && npm run start-dev",
-    "start-dev": "webpack-dev-server --mode development --config=configs/webpack/dev.js --open --open-page \"\"",
+    "start": "concurrently --raw 'npm run clean-dist && npm run start-dev' 'npm run server:dev'",
+    "start-dev": "webpack-dev-server --config=tools/client-webpack/dev.js --open --open-page \"\"",
```

Comment:
rename the scripts specific to client & server

start-dev should start client & server in development mode